datafusion/functions-aggregate/src/min_max.rs

```diff
@@ -610,10 +611,57 @@ fn min_batch(values: &ArrayRef) -> Result<ScalarValue> {
                 min_binary_view
             )
         }
+        DataType::Struct(_) => min_max_batch_struct(values, Ordering::Greater)?,
         _ => min_max_batch!(values, min),
     })
 }
 
+fn min_max_batch_struct(array: &ArrayRef, ordering: Ordering) -> Result<ScalarValue> {
```

Comment:
What definition of "greater / less than" does this use? I think figuring out how to compare structs (like what does it mean for one struct to be bigger than the other) was not clear